articles/azure-maps/elevation-data-services.md

```diff
@@ -0,0 +1,380 @@
+- The [rio-rgbify] Python package, developed by MapBox, is used to encode the GeoTIFF as RGB.
+- [PostgreSQL] database with the [PostGIS] spatial extension.
+
+## Create Contour line vector tiles and RGB-encoded DEM tiles
+
+This guide uses the 36 tiles covering the state of Washington, available from [USGS EarthExplorer].
+
+### Download raster tiles from USGS EarthExplorer
+
+#### Search criteria
+
+Select the region that you want raster tiles for. For demonstration purposes, this guide uses the "Polygon" method to select the region on the map.
+
+1. Navigate to the [USGS EarthExplorer].
+
+1. In the **Search Criteria** tab, select **Polygon** then click on the map to create the boundary.
+
+    :::image type="content" source="./media/elevation-services/create-polygon.png" alt-text="A screenshot showing how to add raster tiles in QGIS." lightbox="./media/elevation-services/create-polygon.png":::
+
+#### Data sets
+
+1. Select the **Data Sets** tab.
+
+1. Select **SRTM 1 Arc-Second Global** from the **Digital Elevations** section.
+
+    :::image type="content" source="./media/elevation-services/data-sets.png" alt-text="A screenshot showing how to add raster tiles in QGIS." lightbox="./media/elevation-services/data-sets.png":::
+
+#### Results
+
+1. Select **Results >>** to view the tiles for the selected region and data set.
+
+1. The list of downloadable tiles appear on the results page. To download
+   only tiles you want, select the **Download Options** button on the result card for each tile,
+   selecting the option **GeoTIFF 1 Arc-Second** and repeat this step for the remaining tiles.
+
+    :::image type="content" source="./media/elevation-services/results-export.png" alt-text="A screenshot showing how to add raster tiles in QGIS." lightbox="./media/elevation-services/results-export.png":::
+
+1. Alternatively, use the bulk download option and select **GeoTIFF 1 Arc-second**.
+
+### Add raster tiles to QGIS
+
+Once you have the raster tiles you need, you can import them in QGIS.
+
+1. Add raster tiles to QGIS by dragging the files to the **QGIS layer**
+   tab or selecting **Add Layer** in the **Layer** menu.
+
+    :::image type="content" source="./media/elevation-services/add-raster-tiles-qgis.png" alt-text="A screenshot showing how to add raster tiles in QGIS." lightbox="./media/elevation-services/add-raster-tiles-qgis.png":::
+
+2. When the raster layers are loaded into QGIS, there can be
+   different shades of tiles. Fix this by merging the raster
+   layers, which result in a single smooth raster image in GeoTIFF
+   format. To do this, select **Miscellaneous** from the **Raster** menu, then **Merge...**
+
+    :::image type="content" source="./media/elevation-services/merge-raster-layers.png" alt-text="A screenshot showing how the merge raster layers menu in QGIS.":::
+
+3. Reproject the merged raster layer to EPSG:3857 (WGS84 / Pseudo-Mercator).
```

Comment:
Please add the "Save Raster Layer as" tool can be accessed from the right click context menu on the layer in table of content > Export > Save As

Reply:
Changed:

> Reproject the merged raster layer to EPSG:3857 (WGS84 / Pseudo-Mercator) using **Save Raster Layer as**
   accessed by right clicking on the merged raster layer in the **table of content** ->
   **Export** -> **Save As** option. EPSG:3857 is required to use it with [Azure Maps Web SDK](https://learn.microsoft.com/azure/azure-maps/about-azure-maps#web-sdk).